src/main/resources/assets/arcanuscontinuum/lang/fr_fr.json

```
@@ -0,0 +1,334 @@
+	"attribute.name.generic.arcanuscontinuum.mana_regen": "Regeneration de mana",
+	"attribute.name.generic.arcanuscontinuum.burnout_regen": "Epuisement de Mana",
+	"attribute.name.generic.arcanuscontinuum.mana_lock": "Réstriction de Mana",
+	"attribute.name.generic.arcanuscontinuum.spell_potency": "Puissance des Sorts",
+  "attribute.name.generic.arcanuscontinuum.mana_cost": "Coût de Mana",
+	"attribute.name.generic.arcanuscontinuum.magic_resistance": "Résistance Magique Rés",
+  "attribute.name.generic.arcanuscontinuum.spell_cool_down": "Refroidissement du sort",
+
+	"config.arcanuscontinuum.spellShapesCategory": "Forme des Sorts",
+	"config.arcanuscontinuum.attackEffectsCategory": "Effet d'Attaque",
+	"config.arcanuscontinuum.supportEffectsCategory": "Effets de Support",
+	"config.arcanuscontinuum.utilityEffectsCategory": "Effets d'utilité",
+	"config.arcanuscontinuum.movementEffectsCategory": "Effets de Mouvements",
+
+	"config.arcanuscontinuum.castingSpeedHasCoolDown": "Utiliser le Refroidissement de l'Attaque quand un Sort est Lancé ?",
+	"config.arcanuscontinuum.sizeChangingIsPermanent": "Size Changing Effects are Permanent for Players and Tamed Animals (Requires Pehkui)",
+	"config.arcanuscontinuum.enabled": "Activé (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.weight": "Poids (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.manaCost": "Coût en Mana (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.manaMultiplier": "Multiplicateur de Mana (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.coolDown": "Temps de Refroidissement (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.minimumLevel": "Niveau Minimum (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.potencyModifier": "Modifieur de Puissance (Requiert un Redémarrage)",
+	"config.arcanuscontinuum.projectileSpeed": "Vitesse du Projectile",
+	"config.arcanuscontinuum.baseLifeSpan": "Temps de Vie de Base",
+	"config.arcanuscontinuum.lifeSpanModifier": "Modifieur du Temps de Vie",
+	"config.arcanuscontinuum.range": "Portée",
+	"config.arcanuscontinuum.delay": "Delai",
+	"config.arcanuscontinuum.strength": "Force",
+	"config.arcanuscontinuum.maximumManaLock": "Verrouillage Maximum du Mana",
+	"config.arcanuscontinuum.maximumAggressorbs": "Maximum d'Aggressorbs",
+	"config.arcanuscontinuum.baseDamage": "Dégâts de Base",
+	"config.arcanuscontinuum.baseTimeOnFire": "Temps en Feu de Base",
+	"config.arcanuscontinuum.baseStunTime": "Durée d'Etourdissement de base",
+	"config.arcanuscontinuum.wetEntityDamageMultiplier": "Multiplicateur de Dégâts contre les Entités Mouillées",
+	"config.arcanuscontinuum.baseFreezingTime": "Temps de Gel de Base",
+	"config.arcanuscontinuum.baseEffectDuration": "Temps d'Effet de Base",
+	"config.arcanuscontinuum.baseHealth": "Vie de Base",
+	"config.arcanuscontinuum.effectDurationModifier": "Multiplicateur de la Durée du Sort",
+	"config.arcanuscontinuum.damageNeededToIncrease": "Dégâts Requits pour Augmenter",
```

```suggestion
	"config.arcanuscontinuum.damageNeededToIncrease": "Dégâts requis pour augmenter la réserve",
```
Took the liberty of adding some context, as the original translation is kind of unclear.